packages/expo-updates/src/Updates.types.ts

```diff
@@ -268,3 +268,29 @@ export enum UpdatesCheckAutomaticallyValue {
  * @hidden
  */
 export type LocalAssets = Record<string, string>;
+
+/**
+ * @hidden
+ */
+export type UpdatesNativeStateMachineContext = {
+  // The native state machine context, either read directly from a native module method,
+  // or received in a state change event. Used internally by this module and not exported publicly.
```

Comment:
move to docblock (putting it here makes tsdoc/vscode think this is for `isUpdateAvailable`)

Reply:
Wow really? I was purposely not putting it in docblock so that it would not appear in docs. I'll do a docs pass

Reply:
I think as long as we have `@hidden` it won't show up, but I could be wrong.

Reply:
Eh, I'm getting mixed results in my vscode. Some of these seem to be associating with the property, others not. Either way, probably best to just put in docblock and keep `@hidden` tag.

Reply:
But I'll leave it up to you on whether you want to put these in docblock. I figure putting them there now is easier since we'll eventually want them there, but could go either way.

Reply:
None of the new methods or types are showing up in generated docs when I run the docs server locally